cli/src/main/java/com/liferay/blade/cli/util/JoinClassLoader.java

```diff
@@ -0,0 +1,115 @@
+package com.liferay.blade.cli.util;
```

Comment:
Need to format this one.  Also need to add a SF exclusion for checking the copyright.